### DISCUSSION TITLE
Separate official and community Simplified Chinese language options

### DISCUSSION THREAD

**Discussion by HZDZ9495** [lonqie/SchaleDB#42] [Translations]
I noticed that the Chinese translation has been recently updated to reflect the CN server version, but the official CN server translation is controversial and many people prefer the original community translation. For instance, the character name "Aru" is translated as "爱露" in the official version, but most people prefer the name "阿露". Using only the official Chinese translations also causes inconvenience for searches, for example "Aru" being translated as "爱露" while "Aru (New Year)" is still translated as "阿露（正月）". I suggest separating the two translations for official cn translation and community cn translation. Thank you to all developers for your hard work.

**Comment by ZhuofanWu** [comment]
Strongly support. The translation by CN servers staffs looks strange. Separate the different translation is a good idea for both servers players.

**Comment by Non-Dalao** [comment]
Can't agree more, I'm upset about the current Simplified Chinese translation :(

**Comment by lonqie** [comment]
Thank you for your feedback. The inconsistency with alt names is indeed an issue, and I will look at getting that resolved so that the names are consistent.
I will consider adding an additional language option that is based on the previous, non-official translation. The official translations will remain as the default option

**Comment by HeartyYF** [comment]
Students like Tomoe who appears in the event plots but haven't been a usable student also have different translations (智惠 vs 巴). If another language option is added, then should these students' names also be adjusted to the "official" edition?

**Comment by lonqie** [comment]
> Students like Tomoe who appears in the event plots but haven't been a usable student also have different translations (智惠 vs 巴). If another language option is added, then should these students' names also be adjusted to the "official" edition?

The official translation will only pull information for released characters, and fall back to the other translation if it is not available. 

I will raise with the person who provides the Chinese translations if they can/want to ensure that character names in the official language option is maintained for CN server unreleased content.
